internal/testrunner/runners/pipeline/test_result.go

```diff
@@ -23,6 +23,25 @@ import (
 
 const expectedTestResultSuffix = "-expected.json"
 
+var geoIPKeys = []string{
+	"as",
+	"geo",
```

Comment:
Not present in ECS, but "geo" is used in packages defined in integrations repository at top level (for instance aws https://github.com/elastic/integrations/blob/b9623f820e24900fdd252cf068012388dec047db/packages/aws/data_stream/s3access/fields/ecs.yml#L25-L26).

I also found usages of "as" at the top level in some packages (for instance netflow https://github.com/elastic/integrations/blob/b9623f820e24900fdd252cf068012388dec047db/packages/netflow/data_stream/log/fields/ecs.yml#L13-L14)


Related to https://github.com/elastic/elastic-package/issues/1435